### ISSUE TITLE
GCE docs issue

### ISSUE THREAD

**Issue by nickpoorman** [mesosphere/mesos-dns#137]
If you look at: [docs/docs/tutorial-gce.md#step-3-launch-mesos-dns](https://github.com/mesosphere/mesos-dns/blob/master/docs/docs/tutorial-gce.md#step-3-launch-mesos-dns)

It tells you to use Marathon to launch Mesos-DNS using the configuration above it, which has port: **`53`** as the bind port. However, Mesos by default only offers ports in the **`31000-32000`** range.

Isn't this going to cause issues with binding? It says those are the defaults under "**Static port mapping**" [here](https://github.com/mesosphere/marathon/blob/master/docs/docs/native-docker.md#bridged-networking-mode)

**Comment by kozyraki** [comment]
@nickpoorman 

In general you are correct, this is not 100% kosher as presented. It relies on the fact that Marathon has no clue that that Mesos-DNS will use port 53 and that marathon/executor/containerizer will not stop Mesos-DNS from doing so, unless the port is already taken on the specific slave. 

There are multiple ways to address this:
- launch Mesos so that it offers port 53 as a resource on (some) slaves. Mesos-DNS/Marathon can require it explicitly
- launch Mesos-DNS in a way that uses bridging so that it's guaranteed to have port 53 available (e.g., docker with bridge mode). Just be careful about IP addresses and port forwarding. 
- launch Mesos-DNS some other way. Many people uses systemd on preselected servers. 

I did not want to make the initial tutorial too complicated so I skipped these. But there have all be tried by us/others recently.
